modules/ossm-about-collecting-ossm-data.adoc

```diff
@@ -31,3 +31,12 @@ $ oc adm must-gather --image=registry.redhat.io/openshift-service-mesh/istio-mus
 ----
 $ oc adm must-gather --image=registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel8:2.4 gather <namespace>
 ----
+
+This creates a local directory that contains the following: 
+
+* The Istio Operator namespace (and its children objects)
```

Comment:
```suggestion
* The Istio Operator namespace and its children objects
```
- Per [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-parentheses), "Avoid using parentheses in general text because text in parentheses is decreased in importance. Include information that is important and eliminate information that is not. Parentheses can also interrupt the flow of a sentence. Try to rewrite the text or use other punctuation instead of parentheses. If you must write a sentence that includes text in parentheses, ensure that the sentence is grammatically correct and technically accurate if you omit the parenthetical text."
- Should "children objects" be "child objects"?